openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/Webhooks/WebhooksPageTest.java

```diff
@@ -108,7 +99,7 @@ void checkDuplicateWebhookName() throws InterruptedException {
     Events.click(webDriver, webhooks.allEntities());
     Events.click(webDriver, webhooks.clickToCloseDropdown());
     Events.click(webDriver, common.saveWebhook());
-    pause(waitTime);
+    Events.waitForElementToDisplay(webDriver, webhooks.toast(), 10);
     WebElement errorMessage = webDriver.findElement(webhooks.toast());
     Assert.assertTrue(errorMessage.isDisplayed());
```

Comment:
Do we need these two lines?

Reply:
@vivekratnavel I have removed the assertions for the same